extensions/amp-autocomplete/0.1/amp-autocomplete.css

```diff
@@ -28,11 +28,16 @@ amp-autocomplete > input {
 .i-amphtml-autocomplete-results {
   position: absolute;
   top: 100%;
-  width: calc(100% + 16px);
+  width: calc(100% + 1rem); /* 100% + input padding-left + 2 * margin-left offset */
+  min-width: calc(2em + 2rem); /* 2 chars. + 2em results padding + 2 * .5em visual offset */
+  max-height: 40vh;
   margin-top: .5rem; /* default-ui-space-small */
   margin-left: -.5rem; /* default-ui-space-small */
   border-radius: 4px; /* default-ui-border-radius */
 
+  overflow-y: auto;
+  overflow-x: hidden;
```

Comment:
When I make an input element narrower in an autocomplete with data that has longer entries, this is a reasonable side effect, right? (The argument being that for datasets where this is unreasonably narrow the input is also unreasonably narrow and you should change that?) Is there a use case where someone would want to specify the width of the options independently from the width of the input?

<img width="369" alt="Screen Shot 2019-04-26 at 10 39 52 AM" src="https://user-images.githubusercontent.com/10456171/56815666-ec506280-680f-11e9-96ef-e898236449a9.png">

Reply:
Yes. In the default case, as well as many custom cases, the length of the result will match the length of the value, and it's usually good practice to make input fields large enough to show their entire value. There ARE some cases where I could have long results (state name + population) and short values (state abbrev.), but that's rare-ish, and I can just set a custom options width.

Which reminds me, actually - what's our CSS customization plan here? Currently developers won't be able to set a custom options width on `i-amphtml-autocomplete-results`. Not launch blocking, but useful.

Also to call out a micro-level decision I made, in the demo, the "favorite state" input has width of 2em, which is a reasonable value for a two character input. BUT it ends up not fitting two characters because it also has `box-sizing: border-box`. I considered taking this off but decided that allowing people to specify `width: 100%` was more important than being able to semantically specify a number of characters. Let me know if you disagree with that tradeoff.

Reply:
Not sure. We don't want open access to the `div.i-amphtml-autocomplete-results` container because that's an opportunity to manually undo default styles, stack them with their custom styles, etc. I think this would fall under the "your styles only or AMP styles only" bucket?